components/Search/SearchForm.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+
+import TextField from 'material-ui/TextField';
```

Comment:
Can we make this a stateless component? Since you're not using any life cycle methods it should be fine. You will need to slightly modify the refs code for that to work, if it's too much of a pain it's ok not to.